files/mcloud/node_modules/@appium/base-driver/build/lib/protocol/protocol.js

```diff
@@ -393,128 +393,131 @@ function buildHandler(app, method, path, spec, driver, isSessCmd) {
-                    throw new Error(`[MCLOUD] HTTP Error ${response.status}`);
-                  }
-              
-                  const responseData = await response.json();
-                  console.log('[MCLOUD] Successful device status change response:', responseData);
-              
-                  return responseData;
+                    const response = await fetch(url, {
+                        method: 'PUT',
+                        headers: headers,
+                        body: JSON.stringify(data)
+                    });
+
+                    if (!response.ok) {
+                        throw new Error(`[MCLOUD] HTTP Error ${response.status}`);
+                    }
+
+                    const responseData = await response.json();
+                    console.log('[MCLOUD] Successful device status change response:', responseData);
+
+                    return responseData;
                 } catch (error) {
-                  console.error('[MCLOUD] Request device status change error:', error.message);
-                  throw error;
+                    console.error('[MCLOUD] Request device status change error:', error.message);
+                    throw error;
                 }
-              }
-              
-              const token = process.env.STF_AUTH_TOKEN 
-              const udid = process.env.DEVICE_UDID              
+            }
+
+            const token = process.env.STF_AUTH_TOKEN
+            const udid = process.env.DEVICE_UDID
+
+
 
 
-            // PATCH #2 <appium>/node_modules/appium/node_modules/appium-base-driver/build/lib/protocol/protocol.js to return to carina "DEBUG info" about problematic step
-            if(err.message !== 'An element could not be located on the page using the given search parameters.'){
-                err.message = `${err.message}[[[--udid ${process.env.DEVICE_UDID} --name ${process.env.DEVICE_NAME} --sessionId ${process.env.sessionId}]]]`;
-            }
-          
             // handling error with Cannot find any free port in range
             if (err.message.toLowerCase().includes("cannot find any free port in range")) {
                 console.log(`[MCLOUD] Cannot find any free port in range. Error message: ${err.message} Appium need restart, exit  code 1 `);
                 // Exiting with code 1 to indicate an error
                 // process.exit(1);
             }
-            
+
             // handling error with ENOSPC: no space left on device, mkdir '/tmp/ and exit with code 0
-            if(err.message.toLowerCase().includes('no space left on device') || err.message.toLowerCase().includes('adb: failed to install')){
+            if (err.message.toLowerCase().includes('no space left on device') || err.message.toLowerCase().includes('adb: failed to install')) {
                 console.log(`[MCLOUD] [BaseDriver] ENOSPC error, no space left on device, Error message:${err.message}. exit code 0. Device is Unhealthy !!! `)
```

Comment:
`Device is Unhealthy !!! `
let's write correctly in english, i.e.
`Device is Unhealthy!`